source/braille.py

```diff
@@ -65,6 +65,8 @@
 import hwIo
 from editableText import EditableText
 
+from speech.extensions import pre_speech, pre_speechCanceled
+

```

Comment:
I don't think we should import from speech at a module level in braille. These two modules should remain independent of each other to reduce cyclical dependency risk

```suggestion
```